src/renderer/components/NavigationBar/NavigationBar.tsx

```diff
@@ -67,7 +68,7 @@ const NavigationBar = () => {
         {renderListOfButtons(BOTTOM_BUTTONS)}
         <button
           className='navbar__button'
-          onClick={() => console.log('TODO: implement logout')}
+          onClick={() => dispatch(terminateSessionAsync())}
```

Comment:
Will this redirect us to the login screen too?

Reply:
yes 👌